src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderMultipleSeparators.java

```diff
@@ -0,0 +1,18 @@
+//non-compiled with javac: contains specially crafted set of imports for testing
```

Comment:
"contains no package declaration"

If we keep this input only to cover single case, can we minimize set of inputs, to keep in file only code that is required for testing, no other. Name of file will change too, and it make more sense.
Sorry that I requested just copy it before.

Reply:
Good point.

Reply:
Inputs improved to cover more cases.

Reply:
The only issue left not fixed I found is 

> An input with empty line in first line and import groups in second line would trigger false positive result.

Some input like this:
```java
//replace this comment with empty line
package com.example; import static java.awt.Button.ABORT; import java.util.Map; class InputCustomImportOrderSingleLine {}
```
with `customImportOrderRules` `STATIC###STANDARD_JAVA_PACKAGE` .

Reply:
it is always better to share problems in format we define in issue template, as it show all details.
your case:
```
$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
  "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
  "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
    <module name="CustomImportOrder">
      <property name="customImportOrderRules"
        value="STATIC###STANDARD_JAVA_PACKAGE"/>
    </module>
  </module>
</module>

$ cat Test.java 
package com.example; import static java.awt.Button.ABORT; import java.util.Map; class InputCustomImportOrderSingleLine {} 

$ java -jar checkstyle-8.24-all.jar -c config.xml Test.java
Starting audit...
[ERROR] /var/tmp/Test.java:1: 'java.util.Map' should be separated from previous import group by one line. [CustomImportOrder]
Audit done.
Checkstyle ends with 1 errors.

# make first line as empty line 
$ cat Test.java 

package com.example; import static java.awt.Button.ABORT; import java.util.Map; class InputCustomImportOrderSingleLine {}$ 

$ java -jar checkstyle-8.24-all.jar -c config.xml Test.java
Starting audit...
Audit done.
```

so it is false-negative (absence of violation), it is much better than false-positive because it will not fail build on user side.
Such code is almost never exists. If somebody write/generate code like this, they highly unlikely need a checkstyle to keep code readable for human. We can try to fix such issue, but the will be no benefit.
I would recommend to ignore such case.